src/Sarif.Converters.UnitTests/PylintConverterTests.cs

```diff
@@ -0,0 +1,154 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Text;
+using FluentAssertions;
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.CodeAnalysis.Sarif.Converters;
+using Microsoft.CodeAnalysis.Sarif.Converters.PylintObjectModel;
+using Moq;
+using Xunit;
+
+namespace Sarif.Converters.UnitTests
+{
+    public class PylintConverterTests
+    {
+        // Pylint is a static code analyzer on python source code.
+        // The source code can be found in https://github.com/PyCQA/pylint
+        // Running Pylint- pylint test_file.py --output-format=json
+
+        private const string InputJson = @"
+        [
+            {
+                ""type"": ""convention"",
+                ""module"": ""kmeans"",
+                ""line"": 21,
+                ""column"": 0,
+                ""path"": ""kmeans.py"",
+                ""symbol"": ""wrong-import-order"",
+                ""message"": ""standard import \""from time import time\"" should be placed before \""from sklearn.datasets import fetch_20newsgroups\"""",
+                ""message-id"": ""C0411""
+            }
+        ]";
+
+        private PylintLogEntry CreateTestPylintLog()
+        {
+            PylintLogEntry PylintLogEntry = new PylintLogEntry()
+            {
+                Type = "convention",
+                ModuleName = "test",
+                Object = "",
+                Line = "1",
+                Column = "120",
+                FilePath = "test.py",
+                Symbol = "testSymbol",
+                Message = "testMessage",
+                MessageId = "C0412"
+            };
+
+            return PylintLogEntry;
+        }
+
+        private PylintLog CreateTestPylintLogList()
```

Comment:
Again, since the type name is now `PylintLog`, the method name would be `CreateTestPylintLog`. But again, there aren't any other kinds of logs in this code, so `CreateTestLog` suffices.